andino_firmware/src/motor.h

```diff
@@ -62,47 +62,44 @@
 // CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
 // OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#ifndef ANDINO_FIRMWARE_SRC_MOTOR_H_
+#define ANDINO_FIRMWARE_SRC_MOTOR_H_
 
-#include "motor_driver.h"
+namespace andino {
 
-#include "Arduino.h"
-#include "commands.h"
-#include "hw.h"
+class Motor {
```

Comment:
Can you add a small description of the class? Just for good practices as the class is simple enough

Reply:
Good catch, done!